packages/fxa-react/components/Portal/index.tsx

```diff
@@ -45,9 +45,12 @@ const Portal = ({
       el.setAttribute('role', 'dialog');
     }
 
-    if (id === 'modal') {
+    if (id === 'dialogs' || id ==='modal') {
```

Comment:
nit: add space after `===`
```suggestion
    if (id === 'dialogs' || id === 'modal') {
```